Documentation/ceph-upgrade.md

```diff
@@ -223,6 +223,16 @@ sed "s/ROOK_NAMESPACE/$ROOK_NAMESPACE/g" upgrade-from-v1.0-create.yaml.tmp > upg
 rm -f upgrade-from-v1.0-create.yaml.tmp
 ```
 
+If you have a v1.0 cluster running with CSI drivers enabled, delete the rbac rules created for CSI
+
+```sh
+kubectl delete -f cluster/examples/kubernetes/ceph/csi/rbac/cephfs/csi-nodeplugin-rbac.yaml
+kubectl delete -f cluster/examples/kubernetes/ceph/csi/rbac/cephfs/csi-provisioner-rbac.yaml
+kubectl delete -f cluster/examples/kubernetes/ceph/csi/rbac/rbd/csi-nodeplugin-rbac.yaml
+kubectl delete -f cluster/examples/kubernetes/ceph/csi/rbac/rbd/csi-provisioner-rbac.yaml
+```
+Note: these files are present in release-1.0 branch
```

Comment:
Instead of telling them the files are in the 1.0 branch, how about if the `kubectl delete` commands point directly to the files from the 1.0 branch? `kubectl` can apply them directly with the link to the raw files. This way they don't need to switch back to the 1.0 branch from the 1.1 branch where they are following the upgrade instructions.